.travis.yml

```diff
@@ -0,0 +1,55 @@
+languge: nix
+matrix:
+  include:
+  - language: nix
+    name: cargo build & linters
+    nix: 2.2.1
+    os: linux
+    script: 'set -e
+
+      nix-shell --run ci_check
+
+      cat $(nix-build ./.travis.yml.nix --no-out-link) > .travis.yml
+
+      git diff -q ./.travis.yml
```

Comment:
What’s this do?

Reply:
It causes the build to fail if .travis.yml xor .travis.yml.nix are changed.